lib/error.js

```diff
@@ -340,6 +341,11 @@ AppError.tooManyRequests = function (retryAfter, retryAfterLocalized) {
     extraData.retryAfterLocalized = retryAfterLocalized
   }
 
+  if (canUnblock) {
+    extraData.verificationMethod = 'email-captcha'
```

Comment:
I specified this name in the doc because that seems to closely match the actual verification method. `email` is already used for `verificationReason: login` and would imply a link needs to be displayed. `unblock-code` doesn't really convey an email is involved and could also imply some other 2fa method, `email-unblock-code` could work, or even just `email-code`. `captcha` seemed to me to imply the user has to type something.